src/main/c/Jep/jep_exceptions.c

```diff
@@ -258,15 +258,15 @@ int process_py_exception(JNIEnv *env)
 
                         // remove the .py to look more like a Java StackTraceElement
                         namelen = (int) strlen(charPyFile);
-                        charPyFileNoExt = malloc(sizeof(char) * (namelen + 1));
+                        charPyFileNoExt = malloc(namelen + 1);
```

Comment:
I like the two sizeof(char) in this here because I had to get sizeof(type) around nearly everything to support more platforms and 32-bit vs 64-bit.  I agree it doesn't make a difference, I just like the style of it.